### ISSUE TITLE
Seg Fault with large diameter

### ISSUE THREAD

**Issue by jogo9225** [Souneth-Ly/Ly_CSCI2270_FinalProject#5]
I'm not sure if this is a "real" problem, but I got a seg fault when I tried to put a diameter in for my created planet in the billions of kilometers.  However, I'm not sure any planets are that big :)

**Comment by Souneth-Ly** [comment]
Yeah if you input the number too big, the program will  crash. We already test that.

**Comment by Jupuu** [comment]
A solution might be to switch the diameter from in to long long and the
stoi to stoll where we switch the diameter from string to int. But instead
now we switch it to a long long.
